Std/Data/Int/DivMod.lean

```diff
@@ -373,6 +373,9 @@ theorem emod_nonneg : ∀ (a : Int) {b : Int}, b ≠ 0 → 0 ≤ a % b
 theorem fmod_nonneg {a b : Int} (ha : 0 ≤ a) (hb : 0 ≤ b) : 0 ≤ a.fmod b :=
   fmod_eq_mod ha hb ▸ mod_nonneg _ ha
 
+theorem fmod_nonneg' (a : Int) {b : Int} (hb : 0 < b) : 0 ≤ a.fmod b := 
+  fmod_eq_emod _ hb.le ▸ emod_nonneg _ hb.ne'
```

Comment:
Change proof to: `  fmod_eq_emod _ (Int.le_of_lt hb) ▸ emod_nonneg _ (Int.ne_of_lt hb).symm`